src/pages/Winners2021.js

```diff
@@ -0,0 +1,369 @@
+                energy grid for power plants by converting a money-losing by-product 
+                (excess energy) into valuable co-products (green hydrogen & oxygen).</Text>
+            
+            <Columns css={`text-align: center`}>
+                <figure>
+                    <Headshot src={Chaitanya} alt="Chaitanya Gulati" css={`width: 10vw; height: 10vw`}/>
+                    <Name>Chaitanya Gulati</Name>
+                </figure>
+                <figure>
+                    <Headshot src={Praneeth} alt="Praneeth Nagarjuna" css={`width: 10vw; height: 10vw`}/>
+                    <Name>Praneeth <br/> Nagarjuna</Name>
+                </figure>
+                <figure>
+                    <Headshot src={Devansh} alt="Devansh Sethia" css={`width: 10vw; height: 10vw`}/>
+                    <Name>Devansh <br/> Sethia</Name>
+                </figure>
+                <figure>
+                    <Headshot src={Khadeeja} alt="Khadeeja Malik" css={`width: 10vw; height: 10vw`}/>
+                    <Name>Khadeeja <br/> Malik</Name>
+                </figure>
+                <figure>
+                    <Headshot src={Dan} alt="Dan Duffy" css={`width: 10vw; height: 10vw`}/>
+                    <Name>Dan Duffy</Name>
+                </figure>
+                <figure>
+                    <Headshot src={Nate} alt="Nate Kraemer" css={`width: 10vw; height: 10vw`}/>
+                    <Name>Nate Kraemer</Name>
+                </figure>
+                <figure>
+                    <Headshot src={Madhu} alt="Madhu Devi" css={`width: 10vw; height: 10vw`}/>
+                    <Name>Madhu Devi</Name>
+                </figure>
+            </Columns>
+
+            <Text><b>LinkedIn:</b><br/>
+            <Link href="https://www.linkedin.com/company/nasadya">https://www.linkedin.com/company/nasadya</Link>
+            </Text>
+
+        </div>
+    )
+}
+
+const Summary = () => {
+    return (
+        <div css={`text-align: center`}>
+            <hr css={`border-top: .2rem solid; 
+                        width: 20vw; 
+                        margin: 4vw auto`} />
+
+            <div css={`margin: 0 10vw`}>
+                <TeamSummary fontSize="0.8rem" fontColor={STEEL_BLUE}>The 2021 PCV Prize drew 87 competing teams from Penn, 
+                Stanford, Columbia, UC Berkeley, U of Illinois, WashU, UTA, UChicago, UToronto, 
+                Cornell, UCSB, Princeton, Yale, Rice, and more.
+                </TeamSummary> <br/> <br/>
+                <TeamSummary fontSize="0.6rem">
+                Winning pitches were chosen from 10 finalist teams at our virtual pitch event by 
+                judges from Fifty Years, Next47, Energy Impact Partners, and Generate Capital:
+                </TeamSummary> <br/>
+            </div>
+
+            <Columns>
+                <ColumnText>
+                    <TeamSummary fontSize="0.6rem" fontColor={STEEL_BLUE}>Waste and Circularity</TeamSummary>
```

Comment:
if every single TeamSummary has the same fontSize do you think you should apply the font-size style at TeamSummary or at every single instance of TeamSummary